{{cookiecutter.project_dashed}}-src/{{cookiecutter.project_namespace}}-config-example.py

```diff
@@ -17,13 +18,12 @@ class DevProfile(object):
 
     MAIL_DEFAULT_SENDER = '{{cookiecutter.developer_email}}'
 
+    KEG_LOG_SYSLOG_ENABLED = False
+
     # Needed by at least KegAuth for sending emails from the command line
     SERVER_NAME = 'localhost:5000'
 
-    CELERY = {
-        # Local rabbitmq server
-        'broker_url': 'amqp://guest@localhost:5672//',
-    }
+    CELERY = celery_config(broker_url='amqp://guest@localhost:5672//')
```

Comment:
If using Docker Compose guidelines per: https://level12.atlassian.net/wiki/spaces/devs/blog/2019/01/25/978092033/Docker+Usage

Then the ports here are wrong.